lib/qiita/markdown/filters/syntax_highlight.rb

```diff
@@ -79,11 +79,11 @@ def has_inline_php?
           end
 
           def highlight(language)
-            Pygments.highlight(code, lexer: language, options: pygments_options)
+            Rouge.highlight(code, language, "html_legacy")
```

Comment:
magic variable may export to variable or methods.

Reply:
I fixed.